examples/spines/spines.py

```diff
@@ -8,6 +8,10 @@
 - normal Axes, with spines on all four sides;
 - an Axes with spines only on the left and bottom;
 - an Axes using custom bounds to limit the extent of the spine.
+
+Each `.axes.Axes` has a list of `~.Spine` objects, accessible
```

Comment:
can this be formatted like the references section in  https://matplotlib.org/stable/gallery/ticks/tick-formatters.html ?

Reply:
I think something like this makes it clearer what the api reference is than having it written out in the prose
![image](https://user-images.githubusercontent.com/1300499/174824228-deee1a50-a48b-4713-8c62-748fafd69023.png)
but reading through again, this might be an in addition to rather than instead of suggestion (sorry!)

Reply:
Sure, that's a good suggestion.  Newest version has that at the bottom now.